clang/lib/Format/TokenAnnotator.cpp

```diff
@@ -3447,9 +3447,13 @@ void TokenAnnotator::calculateFormattingInformation(AnnotatedLine &Line) const {
        Tok = Tok->Next) {
     if (Tok->Previous->EndsCppAttributeGroup)
       AfterLastAttribute = Tok;
-    if (isFunctionDeclarationName(Style.isCpp(), *Tok, Line, ClosingParen)) {
-      LineIsFunctionDeclaration = true;
-      Tok->setFinalizedType(TT_FunctionDeclarationName);
+    if (const bool IsCtorOrDtor = Tok->is(TT_CtorDtorDeclName);
```

Comment:
A honest question, can we assume to have C++17?

Reply:
Yep! We have [moved](https://discourse.llvm.org/t/c-17-in-llvm-code-base/64120/4) to [C++17](https://llvm.org/docs/CodingStandards.html#c-standard-versions) for more than a year.